apps/nextjs-website/src/components/molecules/MarkdownPart/MarkdownPart.tsx

```diff
@@ -0,0 +1,28 @@
+import GitBookContent from '@/components/organisms/GitBookContent/GitBookContent';
+import { Typography } from '@mui/material';
+
+export type MarkdownPartProps = {
+  readonly content: string;
+};
+const MarkdownPart = ({ content }: MarkdownPartProps) => {
+  if (content == null) {
+    return <Typography />; // empty placeholder while loading
+  }
+
+  return (
+    <GitBookContent
+      content={content}
+      config={{
+        isPageIndex: false,
+        pagePath: '',
+        assetsPrefix: '',
+        urlReplaces: {},
+        gitBookPagesWithTitle: [],
+        spaceToPrefix: [],
+      }}
+      hideH1={true}
```

Comment:
```suggestion
      hideLevel1Headings={true}
```
or something similar (you are hiding all occurrences of headings with level 1 and maybe you should use the plural.